docs/reference/index.md

```diff
@@ -44,7 +44,7 @@ ZIO contains a few data types that can help you solve complex problems in asynch
 ## State Management
 
 - **[ZState](state/zstate.md)**— It models a state that can be read from and written to during the execution of an effect.
-- **[Ref](state/ref.md)**— `Ref[A]` models a mutable reference to a value of type `A`.
+- **[Ref](state/global-shared-state.md)**— `Ref[A]` models a mutable reference to a value of type `A`.
```

Comment:
Hey @adamgfraser, thanks for addressing this so fast! 😄 

One suggestion: I know the linked page goes into detail on this, but I think it could be helpful if this short description here could call out the relationship to the `Ref` item described below (and vice versa?). That would definitely help newcomers like me when we're trying to get a feel for all of these ZIO data types at a high level! 🙂

Reply:
@khajavi Can you take a look at this? I think we may just want to delete the reference to `Ref` here since I think it is somewhat confusing to reference it both here and below.